src/main/java/com/chikli/hudson/plugin/naginator/NaginatorActionFactory.java

```diff
@@ -14,13 +14,18 @@
  * @author <a href="mailto:nicolas.deloof@gmail.com">Nicolas De Loof</a>
  */
 @Extension
-public class NaginatorActionFactory extends TransientBuildActionFactory {
+public class NaginatorActionFactory extends TransientActionFactory<AbstractBuild> {
```

Comment:
Review note: `TransientBuildActionFactory` and `TransientActionFactory` is binary incompatible, but I believe OK
(OK means we don't need to change the major version to mark binary incompatible).
I believe external modules never depend on `NaginatorActionFactory`.